Sources/Network/URLSessionNetworkStack.swift

```diff
@@ -8,6 +8,38 @@ public extension Network {
         public typealias Request = URLRequest
         public typealias Response = URLResponse
 
+        public enum Error: Swift.Error {
+
+            public typealias APIError = Swift.Error
+            public typealias TotalRetriedDelay = Retry.Delay
+
+            case noRequest(Swift.Error)
+            case http(HTTP.StatusCode, URLResponse)
+            case api(APIError, HTTP.StatusCode, URLResponse)
```

Comment:
I'm a bit tempted to unify this case again with `.http`, as they're essentially the same (i.e. HTTP protocol error).

On our current project, most errors will be caught in `.api`, but it requires us to always parse an `APIError` from the payload.

However, it makes error handling a bit more cumbersome, because if we want to simply handle errors based on HTTP status code, we always have to check both `.http` and `.api`.

What do you guys think?

Reply:
Well, went ahead with the change, as nobody manifested an opinion on this 😢